drivers/radio/Kconfig

```diff
@@ -0,0 +1,61 @@
+# Kconfig - Radio configuration options
+
+#
+# Copyright (c) 2018 Open Source Foundries Ltd.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig RADIO
+	bool "Radio Drivers"
+	default n
+	help
+	  Enable config options for radio drivers.
+
+if RADIO
+
+config RADIO_WCN14A2A
+	bool "Enable Wistron LTE-M modem driver"
+	select UART_PIPE
+	select NET_OFFLOAD
+	default n
+	help
+	  Choose this setting to enable Wistron WCN14A2A LTE-M modem driver.
+	  NOTE: Currently the pin settings only work with FRDM K64F shield.
+
+config RADIO_WCN14A2A_MAX_MTU
```

Comment:
Is this really an MTU as defined for IP networks? If not, I'd avoid using a well-know, well-define term "MTU". A header below renames this config name as MDM_MAX_DATA_LENGTH. A clean, unambiguous name (MAX_DATA_LEN part I mean). BUF_SIZE is also a good name.